### PR TITLE
feat : added question and result page

### PR DESCRIPTION
**Pull request by BalajiHarinath**
Added question and result page to the quiz app. Please review.

### CODE REVIEW

File: home.css [hunk 1]
```diff
@@ -0,0 +1,126 @@
...
+.footer-social-icons a:hover {
+	color: var(--primary-color);
+}
+
+.footer-header {
+	letter-spacing: 0.5px;
+}
+
+.footer-header span{
+	color: var(--primary-color);
```
Comment (Maianki) [inline home.css:121]: Try keeping your indentation consistent.

File: home.html [hunk 2]
```diff
@@ -0,0 +1,136 @@
...
+
+            <section class="flex flex-column flex-justify-center flex-align-center">
+                <h3  id="Categories">Categories</h3>
+
+                <div class="flex flex-wrap m-2">
+
+                    <div class="category-item flex flex-justify-center flex-align-center" id="oneday">
+                        <div class="quiz-homepage-card flex flex-column m-2 ">
+                            <div class="quiz-homepage-card-img-container">
+                                <img class="quiz-homepage-card-img" loading="lazy" src="./assets/home page images/ODI-worldcup.jpg" alt="image"/>
```
Comment (Maianki) [inline home.html]: Give more meaningful value to the alt attribute.

File: home.html [hunk 3]
```diff
@@ -0,0 +1,136 @@
...
+                                <p class="text-sm">Do you really love OneDay Cricket ? Lets check...</p>
+                                <span class="font-semibold">5 Questions</span>                               
+                            </div>
+                        </div>
+                    </div>
+
+                    <div class="category-item flex flex-justify-center flex-align-center" id="t20"> 
+                        <div class="quiz-homepage-card flex flex-column m-2 ">
+                            <div class="quiz-homepage-card-img-container">
+                                <img class="quiz-homepage-card-img" loading="lazy" src="./assets/home page images/t20-world-cup-trophy.jpg" alt="image"/>
```
Comment (Maianki) [inline home.html]: Here also you can give alt meaningful value.

**Review by Maianki — CHANGES_REQUESTED** [review]
Suggested a few minor changes. Everything else looks good.

File: home.html (hunk 3 above)
Comment (VLeads) [inline home.html]: Don't write in alt tag. Instead give it the name of the image eg. trophy

**Review by VLeads — APPROVED** [review]
Code looks good !

File: home.css (hunk 1 above)
Comment (BalajiHarinath) [reply home.css:121]: I am not sure why the indentation is not correct in git. In vs code the indentation is proper.

File: home.html (hunk 2 above)
Comment (BalajiHarinath) [reply home.html]: sure will change the alt attribute name. Thanks for pointing this out.

File: home.html (hunk 3 above)
Comment (BalajiHarinath) [reply home.html]: Yeah will change this. Thanks for reviewing.

**Review by Maianki — APPROVED** [review]

**Pull request merged**